docs/guide/repository.md

```diff
@@ -438,7 +438,7 @@ This limit is enforced on publishing a new version and only covers archived vers
 
 !!! note "Tech Note"
 
-    Current implementation uses a caching library called [Stash](http://stash.tedivm.com/) (via [StashBundle](https://github.com/tedivm/TedivmStashBundle)). Stash supports the following cache backends: FileSystem, Memcache, APC, Sqlite, Redis and BlackHole.
+    Current implementation uses Symfony cache. It supports the following cache backends: APCu, Array, Chain, Doctrine, Filesystem, Memcached, PDO & Doctrine DBAL, Php Array, Proxy, Redis.
```

Comment:
`It technically supports the following cache backends: [APCu, Array, Chain, Doctrine, Filesystem, Memcached, PDO & Doctrine DBAL, Php Array, Proxy, Redis](https://symfony.com/doc/current/components/cache/cache_pools.html#creating-cache-pools), out of these we recommend use with Redis for cluster and Filesystem for single server.`